test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,76 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import org.junit.Test;
+
+import seedu.addressbook.data.tag.Tag;
+
+public class UtilsTest {
+    private static final Object NULL_OBJECT = null;
+
+    @Test
+    public void isAnyNull() {
+        ArrayList<String> emptyList = new ArrayList<String>();
+        ArrayList<String> nonEmptyList = new ArrayList<String>();
+        nonEmptyList.add("A string");
+
+        // empty list
+        assertFalse(Utils.isAnyNull());
+
+        // Any non-empty list
+        assertFalse(Utils.isAnyNull(new Object(), new Object()));
+        assertFalse(Utils.isAnyNull(emptyList));
+
+        // non empty list with just one at the beginning
+        assertTrue(Utils.isAnyNull(NULL_OBJECT));
+        assertTrue(Utils.isAnyNull(NULL_OBJECT, "", new Object()));
+        assertTrue(Utils.isAnyNull(NULL_OBJECT, emptyList, nonEmptyList));
+
+        // non empty list with nulls in the middle
+        assertTrue(Utils.isAnyNull(nonEmptyList, NULL_OBJECT, NULL_OBJECT, emptyList));
+        assertTrue(Utils.isAnyNull("", NULL_OBJECT, new Object()));
+
+        // non empty list with one null as the last element
+        assertTrue(Utils.isAnyNull("", new Object(), NULL_OBJECT));
+        assertTrue(Utils.isAnyNull(nonEmptyList, emptyList, NULL_OBJECT));
+    }
+
+    @Test
+    public void elementsAreUnique() throws Exception {
+        Tag tag1 = new Tag("tag1"),
+            tag1Copy = new Tag("tag1"),
+            tag2 = new Tag("tag2");
```

Comment:
Should a `commons` test be depending on classes in the ~`model`~ `data` component?

Maybe we should stick with using more basic objects like strings.

If we want to test if `elementsAreUnique()` is calling the `#equals()` and `#hashcode()` methods, then we should use custom stubs for that.